_posts/2024-11-06-#30239.md

```diff
@@ -10,18 +10,43 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Bitcoin Core considers a transaction output to be [dust](https://bitcoin.stackexchange.com/questions/10986/what-is-meant-by-bitcoin-dust) when its value is lower than the cost of creating and spending it.
+
+- Ephemeral dust enables additional options for creating and using dust outputs. This can be useful with types of presigned transactions, chains of transactions, keyed and keyless anchors: allowing zero usage of input funds while still enabling CPFP.
```

Comment:
Perhaps good to ~copy-paste here from the [release notes](https://github.com/bitcoin/bitcoin/blob/40377e810076b639b4cc7a1ddf1488a1e2759432/doc/release-notes-30239.md):

> Ephemeral dust is a new concept that allows a single dust output in a transaction, provided the transaction is zero fee. In order to spend any unconfirmed outputs from this transaction, the spender must also spend this dust in addition to any other outputs.

> Ephemeral dust needs to be both created and spent in the same transaction [package](https://bitcoinops.org/en/topics/package-relay/).

Reply:
Added the first line